WinNSMenu.m

```diff
@@ -151,7 +71,7 @@ void initialize_lock()
 }
 
 // find all subitems for the given items...
-HMENU r_build_menu_for_itemmap(NSMenu *menu, BOOL asPopUp, BOOL fakeItem, NSMapTable *itemMap)
```

Comment:
I think it's a great idea to rename the variable `fakeItem` to `notPullDown` (or something similar) as this value is initially set to `![cell pullsDown]`in `displayPopUpMenu` (line 554).  But I'd suggest we update the name throughout, i.e. rename the `fake` variable in `displayPopUpMenu` and any other method that may use that name?